com.unity.render-pipelines.universal/Runtime/Overrides/Bloom.cs

```diff
@@ -23,6 +23,13 @@ public sealed class Bloom : VolumeComponent, IPostProcessComponent
         [Tooltip("Use bicubic sampling instead of bilinear sampling for the upsampling passes. This is slightly more expensive but helps getting smoother visuals.")]
         public BoolParameter highQualityFiltering = new BoolParameter(false);
 
+        [Tooltip("Skip the smallest bloom pyramid steps.")]
+#if UNITY_ANDROID || UNITY_IOS
+        public ClampedIntParameter skipIterations = new ClampedIntParameter(2, 0, 16);
+#else
+        public ClampedIntParameter skipIterations = new ClampedIntParameter(0, 0, 16);
+#endif
```

Comment:
How does this behave when the user starts with let say a desktop project and changes the player to iOS later on?

Reply:
Currently, the default value is set only when creating the bloom override, so in your example, the skip count would still be at 0. Should that be changed? And if so, what would the expected behavior when changing build targets after changing the value (f.e. change skipIterations to 4 on a desktop project and then change the build target to iOS)

Reply:
That's the question I had, yes. I'm not sure what the expected behavior should be, but I would refrain from changing user values dynamically, that would be too unpredictable and annoying. I'd say either we just set the default to 2 no matter what the platform is, or we leave it at 0 and rely on documentation (risky move).
Overall it's better to have a predictable behavior than one that "tries to do magic" behind the users' back.